Tests/TuistKitTests/Services/TestServiceTests.swift

```diff
@@ -65,7 +65,7 @@ final class TestServiceTests: TuistUnitTestCase {
             XCTAssertEqual(_path, path)
             return workspacePath
         }
-        buildGraphInspector.buildArgumentsStub = { _target, _ in
+        buildGraphInspector.buildArgumentsStub = { _target, _, _ in
```

Comment:
A similar comment here in regards to not ignoring the third argument and adding an assertion.